### PR TITLE
stable release

### PR DESCRIPTION
**Pull request by peppelinux**
I hope to really had well understood what I have to do to push this plugins into official release. Thank you for your support, if something else is missed please let me know in useful times.

### CODE REVIEW

File: net/arp-scan/Makefile [hunk 1]
```diff
@@ -1,8 +1,8 @@
 PLUGIN_NAME=		arp-scan
-PLUGIN_VERSION=		0.1
+PLUGIN_VERSION=		0.2
 PLUGIN_COMMENT=		Get all peers connected to a local network
 PLUGIN_MAINTAINER=	giuseppe.demarco@unical.it
 PLUGIN_DEPENDS=		arp-scan
-PLUGIN_DEVEL=		yes
+PLUGIN_DEVEL=		no
```
Comment (fichtner) [inline net/arp-scan/Makefile]: Needs removal, this is a defined / undefined check

File: net/arp-scan/Makefile [hunk 2]
```diff
@@ -1,8 +1,8 @@
 PLUGIN_NAME=		arp-scan
-PLUGIN_VERSION=		0.1
+PLUGIN_VERSION=		0.2
```
Comment (fichtner) [inline net/arp-scan/Makefile]: And for release 1.0 is recommended :)

Comment (peppelinux) [reply net/arp-scan/Makefile]: Done, sir!

**Comment by peppelinux** [comment]
Ok, done, really!

**Comment by fichtner** [comment]
Merged, thank you!

**Comment by peppelinux** [comment]
Great!

**Comment by fichtner** [comment]
Thank you! :)

**Pull request merged**